<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSet</class>
 <widget class="QWidget" name="SystemSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
color: rgb(255, 255,255);
background-color: rgb(3, 127, 119);
background-color: rgb(2, 80, 85);
background-color: rgb(15, 41, 68);
}
QToolButton{
font: 17pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:1px solid;
border-radius:10px;
border-color: rgb(0, 0, 0);
background-color: rgb(43, 79, 102);
}
QToolButton:hover{
font: 17pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);
color: rgb(0, 185, 185);
/*background-color: rgb(3, 127, 119);*/
}

QPushButton{
font: 14pt &quot;微软雅黑&quot;;
width: 50px;
height:35px;
border:2px solid;
border-radius:8px;
color: rgb(255, 255, 255);
border-color: rgb(0, 170, 127);

}
QPushButton:pressed{
font: 13pt &quot;微软雅黑&quot;;
border-color: rgb(0, 170, 127);
}

QGroupBox{
font: 14pt &quot;微软雅黑&quot;;
border: 1px solid white;
border-radius:8px;
margin-top:10px;
}
QGroupBox:title{
color:white;
subcontrol-origin: margin;
left: 5px;
}
QLabel{
font: 15pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}
QLineEdit{
font: 13pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:2px solid ;
border-color: rgb(0, 185, 185);
border-radius:8px;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>1505</width>
     <height>710</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
border: 1px solid white;
border-radius:8px;
margin-top:10px;
}
QGroupBox:title{
color:white;
subcontrol-origin: margin;
left: 5px;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1485</width>
      <height>635</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTableView {
      selection-background-color: rgb(0, 185, 185);
  }
QTableWidget{
font: 13pt &quot;楷体&quot; rgb(255, 255, 255);
border:1px solid;
gridline-color: rgb(0, 0, 0);
background-color: rgb(15, 41, 68);
}
QCheckBox{
spacing: 10px;
}
QCheckBox::indicator{
width:40px;
height:30px;
margin-left:20px;
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator:unchecked {
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator:checked {
image: url(:/Image/checked.png);
}
</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnDel">
    <property name="geometry">
     <rect>
      <x>1285</x>
      <y>20</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEditHostIP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>175</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000.000.000.000</string>
    </property>
    <property name="text">
     <string>192.168.126.245</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnAdd">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>20</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnSave">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>20</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEditHostAddress">
    <property name="geometry">
     <rect>
      <x>885</x>
      <y>20</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>监控主机的地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>主机IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>20</y>
      <width>96</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>安装地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>主机端口号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEditHostPort">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>20</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>主机端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>96</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>节点数目:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEditNodeNum">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>20</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>节点数目</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
